hypershift-operator/controllers/hostedcluster/metrics/metrics.go

```diff
@@ -472,17 +480,26 @@ func (c *hostedClustersMetricsCollector) Collect(ch chan<- prometheus.Metric) {
 				}
 
 				if metricLabels["cluster_type"] == "rosa" {
-					etcdRecoveryActiveCondition := meta.FindStatusCondition(hcluster.Status.Conditions, string(hyperv1.EtcdRecoveryActive))
-					if etcdRecoveryActiveCondition.Status == metav1.ConditionFalse && etcdRecoveryActiveCondition.Reason == hyperv1.EtcdRecoveryJobFailedReason {
+					etcdRecoveryActiveCondition := meta.FindStatusCondition(hcluster.Status.Conditions, string(hyperv1beta1.EtcdRecoveryActive))
+					if etcdRecoveryActiveCondition.Status == metav1.ConditionFalse && etcdRecoveryActiveCondition.Reason == hyperv1beta1.EtcdRecoveryJobFailedReason {
 						etcdManualInterventionRequiredValue := 1.0
 						ch <- prometheus.MustNewConstMetric(
 							etcdManualInterventionRequiredMetricDesc,
 							prometheus.GaugeValue,
 							etcdManualInterventionRequiredValue,
-							append(hclusterLabelValues, metricLabels["rosa_environment"], metricLabels["rosa_id"])...,
+							append(hclusterLabelValues, metricLabels["environment"], metricLabels["id"])...,
 						)
 
 					}
+
+					if sizeOverride := hcluster.Annotations[hyperv1beta1.ClusterSizeOverrideAnnotation]; sizeOverride != "" {
+						ch <- prometheus.MustNewConstMetric(
+							clusterSizeOverrideMetricDesc,
+							prometheus.GaugeValue,
+							1.0,
+							append(hclusterLabelValues, metricLabels["environment"], metricLabels["id"], sizeOverride)...,
```

Comment:
**Question:** we are already adding an `_id` label, how does that differ from the `id` here? It might be confusing to have an `_id` label as well as `id`.

Refs:
- https://github.com/openshift/hypershift/blob/37db51a9793e78bdf3a7621e958f605fe1f04c84/hypershift-operator/controllers/hostedcluster/metrics/metrics.go#L330
- https://github.com/openshift/hypershift/blob/37db51a9793e78bdf3a7621e958f605fe1f04c84/hypershift-operator/controllers/hostedcluster/metrics/metrics.go#L129

Reply:
as agreed offline, we're keeping `_id` and will name other one `internal_id`